bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap.properties

```diff
@@ -76,6 +76,8 @@ thing-type.openweathermap.weather-api.description = Provides access to the OpenW
 
 bridge-type.config.openweathermap.weather-api.apiVersion.label = One Call API Version
 bridge-type.config.openweathermap.weather-api.apiVersion.description = One Call API version (defaults to 2.5, version 3.0 is available, but needs different subscription).
+bridge-type.config.openweathermap.weather-api.apiVersion.option.2.5 = 2.5
```

Comment:
Even though these lines are generated by the i18n tool, they are not needed, so please remove them after regenerating the i18n files (final step after changing any of the .xml files and to do before committing and pushing):

```suggestion
```